apps/user/src/components/common/Header/Header.tsx

```diff
@@ -84,22 +84,23 @@ const Header = () => {
 export default Header;
 
 const StyledHeader = styled.div`
-    max-width: 1448px;
+    width: 100%;
     height: 118px;
     background-color: ${color.white};
-    margin: 0 auto;
-    padding: 0px 100px;
-    border-bottom: 1px solid ${color.gray200};
 `;
 
 const HeaderBar = styled.div`
     ${flex({ alignItems: 'center', justifyContent: 'space-between' })}
+    padding: 0px 100px;
     width: 100%;
+    height: 64px;
 `;
 
 const NavigationBar = styled.div`
     ${flex({ alignItems: 'center' })}
     width: 100%;
+    padding: 0px 96px;
     height: 54px;
     background-color: ${color.white};
+    border-bottom: 1px solid ${color.gray200};
 `;
```

Comment:
여기 코드 다시 짜주세요

```suggestion
const StyledHeader = styled.div`
    width: 100%;
    height: 118px;
    background-color: ${color.white};
    border-bottom: 1px solid ${color.gray200};
`;

const HeaderBar = styled.div`
    ${flex({ alignItems: 'center', justifyContent: 'space-between' })}
    max-width: 1248px;
    margin: 0 auto;
    padding: 0px 100px;
    height: 64px;
`;

const NavigationBar = styled.div`
    ${flex({ alignItems: 'center' })}
    max-width: 1248px;
    margin: 0 auto;
    padding: 0px 96px;
    height: 54px;
`;
```
하고 UnderlineButton 백그라운드 지우면

<img width="1191" alt="스크린샷 2023-08-28 오후 9 07 14" src="https://github.com/Bamdoliro/marururu/assets/102217839/cd4dfc00-53e8-4184-8a5a-4b368fe2f026">

이렇게 디자인데로 딱 되는데 어떠신가요?

Reply:
UnderlineButton의 백그라운드를 지우고 보더를 보이게하는것이 좋은 방법인지는 생각해봐야할것같아요